<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>koszyk</class>
 <widget class="QWidget" name="koszyk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>koszyk</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>441</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Paragon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Faktura</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>421</width>
       <height>191</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ImiÄ™</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="imie_txt"/>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nazwisko</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nazw_txt"/>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>NIP</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nip_txt"/>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="Zatwierdz_klient_btn">
       <property name="text">
        <string>Zatwierdz</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Anuluj_btn2">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>431</width>
       <height>371</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTableWidget" name="produkty_table"/>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="zatwierdz_btn">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
      <widget class="QPushButton" name="anuluj_btn">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
